doc/ingest.md

```diff
@@ -86,6 +88,35 @@ The network indexer nodes expect that metadata begins with a `uvarint` identifyi
   * the proposed `uvarint` protocol is `0x3D0000`.
   * the following bytes are not yet defined.
 
+#### ExtendedProvider
+
+The `ExtendedProvider` field allows for specification of provider families, in cases where a provider operates multiple PeerIDs, perhaps with different transport protocols between them, but over the same database of content.
+
+```
+type ExtendedProvider struct {
+    Providers [Provider]
+    Override bool
+}
+
+type Provider struct {
+    ID String
+    Addresses [String]
+    Metadata optional Bytes
+    Signature Bytes
+}
+```
+
+* If `Metadata` is not specified for a `Provider`, the metadata from the encapsulating  `Advertisement` will be used instead.
+* If a `Provider` listing is written with no `ContextID`, those peers will be returned for all advertisements published by the publisher.
+  * If `Override` is set on an `ExtendedProvider` entry on an advertisement with a `ContextID`, it indicates that any specified chain-level set of providers should not be returned for that context ID. `Providers` will be returned Instead.
```

Comment:
This seems like it overlaps in functionality with a removal (`IsRm`) advertisement for provider and context ID.  If that is the case, then maybe it should be more explicit (set the `IsRm` flag) to specify that a context ID is being deleted for a set of providers.  Or, is there a different use case for this?

In any case, the semantics of `ExtendedProvider` in a removal advertisement should be specified.

Reply:
added line indicating `ExtendedProviders` should be ignored in `IsRm` advertisements.